src/metabase/core.clj

```diff
@@ -31,7 +34,16 @@
              [params :refer [wrap-params]]
              [session :refer [wrap-session]]]
             [toucan.db :as db])
-  (:import org.eclipse.jetty.server.Server))
+  (:import java.util.Locale
+           org.eclipse.jetty.server.Server))
+
+;; HACK: replace this with setting event listener
+(defn sync-locale []
+  (let [new-locale     (or (not-empty (setting/get :site-locale)) "en")
+        current-locale (.toLanguageTag (Locale/getDefault))]
+    (log/info (trs "Hello World!"))
+    (if (not= current-locale new-locale)
+      (Locale/setDefault (Locale/forLanguageTag new-locale)))))
```

Comment:
I think you'd usually see a `when-not` here, or `(when (not...`

Reply:
👍